### ISSUE TITLE
Log tracebacks, add errored upload file for test

### ISSUE THREAD

**Issue by shubham0704** [shubham0704/tutor.ai#2]
In order to make the preprocessing robust. We need to log the trace backs and add the files for testing directly

Deliverables - 
+ Log tracebacks into a logs folder along with session details.
+ Add error file to ai_tutor/test/test_files/

We can then schedule sessions for hourly debugging.

**Comment by Souldiv** [comment]
since the file structure will be different for ai_tutor in deployment, do you want me to create a separate  folder test in tutor_ai and add the uploaded file there?

**Comment by shubham0704** [comment]
Yes, that would be great. along with the traceback. That can be automatically get stored if we use a logger.

**Comment by Souldiv** [comment]
logging tracebacks done, have to save uploaded files.
